src/app/products/manage-tax-components/create-tax-component/create-tax-component.component.html

```diff
@@ -0,0 +1,71 @@
+<div class="container">
+
+  <mat-card>
+
+    <form [formGroup]="taxComponentForm" (ngSubmit)="submit()">
+
+      <mat-card-content>
+
+        <div fxLayout="column">
+
+          <mat-form-field>
+            <mat-label>Name</mat-label>
+            <input matInput required formControlName="name">
+            <mat-error *ngIf="taxComponentForm.controls.name.hasError('required')">
+              Name is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Percentage</mat-label>
+            <input type="number" matInput required formControlName="percentage">
+            <mat-error *ngIf="taxComponentForm.controls.percentage.hasError('required')">
+              Percentage is <strong>required</strong>
+            </mat-error>
+            <mat-error *ngIf="taxComponentForm.controls.percentage.hasError('min')">
+              Percentage <strong>cannot</strong> be smaller than zero
+            </mat-error>
+          </mat-form-field>
+
```

Comment:
Please add the fields for Debit Account Type and Debit Account as well: https://github.com/openMF/community-app/blob/develop/app/views/products/tax/CreateTaxComponent.html 
Ensure they are not hidden in the web app.

Make appropriate changes to show Credit Account Field as mandatory if Credit Account Type is selected. Similar case should be handled for Debit Account as well.